app/javascript/javascript/filtersUtils.js

```diff
@@ -74,21 +74,44 @@ export const setupTimeRangeFilter = (
     );
   }
 };
-export const setupAutocomplete = (callback, id, path, createParams) => {
-  if (document.getElementById(id)) {
-    $(`#${id}`).autocomplete({
+export const setupTagsAutocomplete = (callback, path, createParams) => {
+  if (document.getElementById("tags")) {
+    $(`#tags`)
+      .autocomplete({
+        source: function(request, response) {
+          const data = {
+            q: { input: request.term, ...createParams() }
+          };
+          $.getJSON(path, data, response);
+        },
+        select: function(event, ui) {
+          callback(ui.item.value);
+        },
+        minLength: 0
+      })
+      .focus(function() {
+        $(this).autocomplete("search");
+      });
+  } else {
+    window.setTimeout(setupAutocomplete(callback, path), 100);
+  }
+};
+
+export const setupProfileNamesAutocomplete = callback => {
+  if (document.getElementById("profile-names")) {
+    $(`#profile-names`).autocomplete({
```

Comment:
how about using `document.getElementById("profile-names")` here as well, possibly keeping it in a variable above the if?

Reply:
then we could get rid of jquery in one more place :)

Reply:
Doesn't work. This library only works with elements found by jquery. Cause `Uncaught TypeError: document.getElementById(...).autocomplete is not a function`

Reply:
😿